fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -1735,4 +1735,33 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         key = appContext.getPreferenceKey(R.string.pref_key_enable_tabs_tray_to_compose),
         default = FeatureFlags.composeTabsTray,
     )
+
+    /**
+     * Adjust Activated User sent
+     */
+    var growthUserActivatedSent by booleanPreference(
+        key = appContext.getPreferenceKey(R.string.pref_key_growth_user_activated_sent),
+        default = false,
+    )
+
+    /**
+     * Indicates how many days in the first week user opened the app.
+     */
+    val growthEarlyUseCount = counterPreference(
+        appContext.getPreferenceKey(R.string.pref_key_growth_early_browse_count),
+        maxCount = 2,
```

Comment:
Should this `maxCount` be 3 or perhaps just use the default argument? Just feels a little weird to have a discrepancy between this and `daysActivatedThreshold`. It doesn't look like this property is actually used for anything in this case so I don't think it matters much, but it either update might be a more clear signal to future readers.